docs/source/tutorials_source/platform/tutorial_sunflowers.py

```diff
@@ -1,56 +1,54 @@
 """
 
-Tutorial 2: Sample Sunflowers
+Tutorial 2: Diversify your Dataset using Coreset
 =============================================
 
 This tutorial highlights the basic functionality of sampling in the web-app.
+You can use the Coreset sampling algorithm to choose a diverse subset of your dataset.
+This can be useful many purposes, e.g. for having a good subset of data to label
+or for creating a validation or test dataset that covers the complete sample space.
+Removing duplicate images can also help you in reduce bias and imbalances in your dataset.
 
 What you will learn
 --------------------
 
-* Upload images to the web-app via frontend or Python package
-* Upload embeddings to the web-app via frontend or the Python package
+* Upload images and embeddings to the web-app via the Python package
 * Sample a diverse subset of your original dataset in the web-app
+* Download the filenames of the subset and use it to create a new local dataset folder.
 
 Requirements
 -------------
 You can use your own dataset or the one we provide for this tutorial. The dataset
-we provide consists of 734 images of sunflowers and an embedding file. You can
+we provide consists of 734 images of sunflowers. You can
 download it here :download:`Sunflowers.zip <../../../_data/Sunflowers.zip>`.
 
-You can also use our pre-trained models to create your own embeddings:
+To use the Lightly platform, we need to upload the dataset with embeddings to it.
+The first step for this is to train a self-supervised embedding model.
+Then, embed your dataset and lastly, upload the dataset and embeddings to the Lightly platform.
+These three steps can be done using a single command from the lightly pip package: lightly-magic
 
```

Comment:
I would maybe add one more sentence here, e.g. "But first, we need to install lightly from the Python package index."